packages/legacy-scripting-runner/test/example-app/index.js

```diff
@@ -1171,6 +1183,19 @@ const App = {
       source: "return z.legacyScripting.run(bundle, 'hydrate.file');"
     }
   },
+  // This breaks `applyBeforeMiddleware` which it looks like gets the original definition without `findSourceRequireFunctions` applied?!
+  // beforeRequest: [
```

Comment:
I found a bug but wasn't able or required to fix it now.

Reply:
Nice finding! Looks like it's an issue with the test code. Those failing tests don't compile the app definition but they should. I'll push a fix once this is merged.